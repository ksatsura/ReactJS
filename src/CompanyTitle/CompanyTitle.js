// import
import React from 'react';
import '../style.css';

const CompanyTitle = () => {

    return <p className='company-title'>netflixroulette</p>;
};

export default CompanyTitle;