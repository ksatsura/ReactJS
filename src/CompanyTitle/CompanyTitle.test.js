import React from 'react';
import CompanyTitle from './CompanyTitle';
import {shallow} from 'enzyme';

describe('CompanyTitle.js', () => {

    describe('renders', () => {

        it('renders for desktop', () => {
            const component = shallow(<CompanyTitle />);
            expect(component).toMatchSnapshot();
        });
    });
});
