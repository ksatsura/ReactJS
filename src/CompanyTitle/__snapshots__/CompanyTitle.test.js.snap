// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyTitle.js renders renders for desktop 1`] = `ShallowWrapper {}`;

exports[`CompanyTitle.js renders renders for mobile 1`] = `ShallowWrapper {}`;
